### PR TITLE
[Flatbuffer] Add flatbuffer_opnode @open sesame 03/13 16:13

### PR DESCRIPTION
**Pull request by DonghakPark**
## Add flatbuffer_opnode.cpp & flatbuffer_opnode.h
- For flatbuffer interpreter : bring opnode and make variable
- Now only support Fully Connected Layer
- Now only support NCHW format tensor

**_It will be updated_**

Signed-off-by: DongHak Park <donghak.park@samsung.com>

### CODE REVIEW

**Review by myungjoo — APPROVED** [review]

**Review by baek2sm — APPROVED** [review]
LGTM

**Review by SeoHyungjun — APPROVED** [review]
LGTM!

**Review by songgot — APPROVED** [review]
LGTM

File: nntrainer/compiler/flatbuffer_opnode.h [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+   * @return const nntr::BuiltinOptions
+   */
+  const nntr::BuiltinOptions getOptionType() const {
+    return builtin_option_type;
+  }
+
+  /**
+   * @brief Get the Builtin Ops object
+   * @param f Flatbuffer builder
+   * @return flatbuffers::Offset<void>
```
Comment (jihochu) [inline nntrainer/compiler/flatbuffer_opnode.h:150]: empty line?

File: nntrainer/compiler/flatbuffer_opnode.cpp [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+namespace nntrainer {
+
+FlatBufferOpNode::FlatBufferOpNode() :
+  inputs(),
+  outputs(),
+  weights(),
+  is_input(),
+  is_output(),
+  is_virtual(),
```
Comment (jihochu) [inline nntrainer/compiler/flatbuffer_opnode.cpp:27]:
```suggestion
  is_virtual(false),
```
?? what is default value?

Comment (DonghakPark) [reply nntrainer/compiler/flatbuffer_opnode.cpp]: [FIX] Thank you for Review

File: nntrainer/compiler/flatbuffer_opnode.cpp [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+#include <layer_context.h>
+#include <layer_node.h>
+#include <memory.h>
+
+namespace nntrainer {
+
+FlatBufferOpNode::FlatBufferOpNode() :
+  inputs(),
+  outputs(),
+  weights(),
```
Comment (jijoongmoon) [inline nntrainer/compiler/flatbuffer_opnode.cpp:24]: Do we need this initialization?

Comment (DonghakPark) [reply nntrainer/compiler/flatbuffer_opnode.cpp:24]: [Fix] Delete unnecessary initialization

**Pull request merged**